### ISSUE TITLE
register_tag functionality?

### ISSUE THREAD

**Issue by savraj** [pwwang/liquidpy#18]
Is this feature supported by liquidpy? The ability to register_tags?

https://github.com/Shopify/liquid/wiki/Liquid-for-Programmers#create-your-own-tags

My issue is that I'm rendering a template that has a couple tags I want to ignore -- the template is also used in a jekyll site as well -- but I'm also rendering some pages with python.

This doesn't work when I set the variable `python` to `True` because I think `liquidpy` is scanning all the tags before it processes the `unless`.

```
    {% unless python %}
      {% feed_meta %}
      {% seo %}
    {% endif %}
```
I'm trying to map `{% feed_meta %}` and `{% seo %}` to `''`.

**Comment by pwwang** [comment]
This feature is not yet supported, but I will keep this open and explore it to see if it can be implemented in the next major version.

For your later question, `liquidpy` is scanning all the tags before processing them, not only `unless`, but all of them. I have a little experiment on your code:

```python console
Python 3.7.4 (default, Aug 13 2019, 20:35:49)
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from liquid import Liquid
>>> liq = Liquid("""
... {% unless python %}
...     {{ feed_meta }}
...     {{ seo }}
... {% endunless %}
... """)
>>> liq.render(python=True, feed_meta="Some feed meta", seo="Some seo")
'\n\n'
>>> liq.render(python=False, feed_meta="Some feed meta", seo="Some seo")
'\n\n    Some feed meta\n    Some seo\n\n'
```

When `python` is `True`, the block inside is not rendered. Is this what you expected?

**Comment by savraj** [comment]
Yes that is the expected behavior you show, the problem is in my example feed_meta is surrounded by `{% and %}` which causes the issue. Understood though! Thank you.

**Comment by savraj** [comment]
@pwwang I found a solution -- I wrote a template file that works with both liquid AND jinja, so I use jinja for extra features I need. :) So you can close this and the other issue if you like!

**Comment by pwwang** [comment]
Great! Glad you had this cool thing invented! I am working on remodeling the nodes so that developers can register their own nodes.
I will close this once I publish that version.
Thank you!

**Comment by pwwang** [comment]
Addressed in `v0.5.0`

https://liquidpy.readthedocs.io/en/latest/nodes/#write-you-only-statement-node
